[ipinfo] isTimeSensitive (#34150)
demisto/content: Packs/ipinfo/ReleaseNotes/2_1_22.md

#### Integrations

##### IPinfo v2

> Updated the ***ip*** command when used for auto enrichment, the command timeout is reducing to 2 seconds.

Reviewer: Let's discuss it with a technical writer

> Updated the ***ip*** command, reducing the timeout of 2 seconds when used for auto enrichment using `auto-extract=inline`.

Reviewer 2: ```suggestion
Updated the ***ip*** command, reducing the timeout from 20 seconds to 2 seconds when used for auto enrichment using `auto-extract=inline`.
```

> Updated the ***ip*** command when used for auto enrichment with auto-extract=inline, the command timeout is reducing to 2 seconds.

Reviewer: ```suggestion
Updated the ***ip*** command, reducing the timeout of 2 seconds when used for auto enrichment using `auto-extract=inline`.
```
